packages/fx-core/templates/plugins/resource/hosting/bicep/function/config.template.bicep

```diff
@@ -0,0 +1,12 @@
+// Get existing app settings for merge
```

Comment:
Nit: A interesting question came to my mind - do we need to add MIT copyright to these Bicep files?

Reply:
No. Bicep files are user files of a sort. We should not add any copyright for them.